docs/dev/table/sql/queries.md

```diff
@@ -313,9 +313,29 @@ SHOW DATABASES;
 		<p>Show all tables in the current database in the current catalog</p>
 {% highlight sql %}
 SHOW TABLES;
+{% endhighlight %}
+        <p>Show all views in the current database in the current catalog</p>
+{% highlight sql %}
+SHOW VIEWS;
 {% endhighlight %}
       </td>
     </tr>
+    <tr>
+      <td>
+        <strong>Describe</strong><br>
+        <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span>
+      </td>
+      <td>
+			<p>Describe the given table.</p>
```

Comment:
explain more about `Describe` operation ? such as `Describe the schema of the given table.`